lib/hashes.md

```diff
@@ -5,40 +5,46 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a Hash?
 
-Replace me with your response...
+A hash is a group of items in which each item has an attribute or value stored along with it.
```

Comment:
1) race_results = {
"winner" => "Mindy",
"runner-up" => "Daniel",
"bronze" => "Sumeet"
}
2) The words in the 'item' column are called 'keys', and the right column contains the 'values' of the keys.